src/sdk/accountspace_fs.js

```diff
@@ -26,6 +27,18 @@ const { account_cache } = require('./object_sdk');
 const dummy_region = 'us-west-2';
 const dummy_service_name = 's3';
 
+const account_id_cache = new LRUCache({
```

Comment:
We already had imported the `account_cache` from object sdk, what is the difference between `account_id_cache` and `account_cache`?

Reply:
The difference is -  
 - Regular accounts cache - access_key -> account mapping
 - new Accounts ID Cache id -> account mapping.